asistente_ladm_col/logic/quality_rules/qr_plots_covered_by_boundaries.py

```diff
@@ -0,0 +1,483 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-07-05
+        git sha         : :%H$
+        copyright       : (C) 2022 by Leo Cardona (SwissTierras Colombia)
+                          (C) 2021 by Germán Carrillo (SwissTierras Colombia)
+        email           : contacto@ceicol.com
```

Comment:
Order the authors ;)

Reply:
Fixed in [9570995](https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/9570995dcec2a4d6948b234031e35ba697c5f934)